app/ui/pages/10_Settings.py

```diff
@@ -20,7 +22,7 @@ def invalid_number(string: str) -> Optional[str]:
 
 
 def get_task_status(dbname):
-    sql = f"""call admin.run_as_app('describe task TASKS."{dbname}";');"""
+    sql = f"""describe task TASKS."{dbname}";"""
```

Comment:
I'm surprised that we had this wrapped in the `run_as_app`. Do you remember if there was a reason we had this one?

Reply:
I don't think tasks were exposed to admin in early iterations